### PR TITLE
Windows attempt

### PR DESCRIPTION
**Pull request by bluescarni**

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -15,12 +15,11 @@ source:
     - ambiguous_nan_inf.patch   # [linux]
 
 build:
-  number: 0
+  number: 1
   features:
     - vc9     # [win and py27]
     - vc10    # [win and py34]
     - vc14    # [win and py35]
```
Comment (isuruf) [inline recipe/meta.yaml]: Change this to `# [win and py>=35]`

File: recipe/meta.yaml [hunk 2]
```diff
@@ -15,12 +15,11 @@ source:
...
-  skip: true  # [win]
```
Comment (isuruf) [inline recipe/meta.yaml]: Skip python 3.6 on windows as both 3.5 and 3.6 use the same msvc version

**Comment by bluescarni** [comment]
@isuruf Thanks for the review!

I have also disabled the py27 win64 build as the tests don't run there for some reason.

File: recipe/bld.bat [hunk 3]
```diff
@@ -16,6 +16,12 @@ IF "%CONDA_PY%"=="35" (
     SET ENABLE_CXX11=NO
 )
 
+IF "%CONDA_PY%"=="36" (
```
Comment (isuruf) [inline recipe/bld.bat]: Shouldn't this be "35"?

Comment (bluescarni) [reply recipe/bld.bat]: I think this is redundant now, as above it's already checking for 35. I'll remove it.

**Comment by isuruf** [comment]
The line `IF "%CONDA_PY%"=="27" SET GENERATOR=NMake Makefiles` can also be removed as we've fixed the issue in conda-forge

**Comment by isuruf** [comment]
Can you try win64 python 2.7 now after the latest commit f2a9540

**Comment by bluescarni** [comment]
@isuruf good to go?

**Comment by isuruf** [comment]
Segfaults on windows. Not sure why the build is green.

**Comment by bluescarni** [comment]
Ah damn. I wonder if that MAX_VARIADIC param here has any effect:

https://github.com/msgpack/msgpack-c/blob/master/appveyor.yml

**Pull request merged**